### DISCUSSION TITLE
Problem in Deployment - This request has been blocked; the content must be served over HTTPS

### DISCUSSION THREAD

**Discussion by otavio-araujo** [filamentphp/filament#3056] [Help]
Im trying to deploy on Digitam Ocean but, the page gets bank and the console gets this message:

Mixed Content: The page at 'https://heradesign-2lmif.ondigitalocean.app/' was loaded over HTTPS, but requested an insecure script 'http://heradesign-2lmif.ondigitalocean.app/filament/assets/app.js?id=b0614ef3185ae0741d9e52d09eba7fac'. This request has been blocked; the content must be served over HTTPS.

I've already tryed all solutions that i could found on google about this error, bur, unfortunatly unsucceeded ones.

Does anyone know how to solve this?

**Comment by danharrin** [comment]
1) Check your `APP_URL`
2) Use `URL::forceScheme('https')` in a service provider.

**Reply by otavio-araujo** [reply]
I did, but it does not work.

![image](https://user-images.githubusercontent.com/30505243/177852278-4542bdb5-9841-4763-8360-95bdb41bd36d.png)

here's my AppServiceProvider

![image](https://user-images.githubusercontent.com/30505243/177852529-4152ef8b-40c3-4a78-8d62-f9890869e5ba.png)

With the use Illuminate\Support\Facades\URL; on the top.

**Reply by danharrin** [reply]
Have you allowed all proxies in the TrustProxies middleware?

**Comment by otavio-araujo** [comment]
Yeah... I Did that too...

I'll try another host...

Thank you!

Atenciosamente,

Otávio Araújo.

ELITE - Empresa de Limpeza e Tratamentos Especiais

Fone: (44) 3030-0655
WhatsApp: (44) 99119-0655
________________________________
De: Dan Harrin ***@***.***>
Enviado: Thursday, July 7, 2022 5:54:47 PM
Para: filamentphp/filament ***@***.***>
Cc: Otávio Araújo ***@***.***>; Author ***@***.***>
Assunto: Re: [filamentphp/filament] Problem in Deployment - This request has been blocked; the content must be served over HTTPS (Discussion #3056)


Have you allowed all proxies in the TrustProxies middleware?

**Comment by hrvoj3e** [comment]
When using behind proxy `URL::forceScheme('https')` solves this problem.

**Comment by cpbret** [comment]
theme(asset('css/filament/app/theme.css'))

->

theme(secure_asset('css/filament/app/theme.css'))

was the answer for us

**Reply by kecapjeruknipis** [reply]
this fix my problem on filament v3

**Reply by yazidr1** [reply]
where to add those line ? in the root layout, normally app.blade in layout ?

**Reply by SalcidoJesus** [reply]
> where to add those line ? in the root layout, normally app.blade in layout ?

In my case, I add this in my **AdminPanelProvider.php**, I had a error showing my favicon, but I solved with this
`->favicon(secure_asset('favicon.ico'))`
